addons/stock/wizard/stock_picking_return.py

```diff
@@ -43,17 +45,18 @@ def default_get(self, fields):
         'stock.location', 'Return Location',
         domain="['|', ('id', '=', original_location_id), '|', '&', ('return_location', '=', True), ('company_id', '=', False), '&', ('return_location', '=', True), ('company_id', '=', company_id)]")
 
-    @api.onchange('picking_id')
-    def _onchange_picking_id(self):
+
+    def _compute_moves_locations(self, vals):
```

Comment:
`_compute*` functions are usually reserved for actual compute functions, so try to rename it, check: https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#symbols-and-conventions